scipy/stats/_wilcoxon.py

```diff
@@ -111,15 +111,6 @@ def _wilcoxon_iv(x, y, zero_method, correction, alternative, method, axis):
     if method == "auto" and d.shape[-1] > 50:
         method = "asymptotic"
 
-    if n_zero > 0 and method == "exact":
-        warnings.warn("Exact p-value calculation does not work if there are "
-                      "zeros. Consider using method `asymptotic` or "
-                      "`stats.PermutationMethod`",
-                      stacklevel=2)
-
-    if 0 < d.shape[-1] < 10 and method == "asymptotic":
-        warnings.warn("Sample size too small for normal approximation.", stacklevel=2)
```

Comment:
Similarly here: does this imply that a sample size of 10 is large enough for a normal approximation?

Note that there wasn't actually a test for this warning before. It was always just filtered out, so presumably it was not deemed to be critical.